### DISCUSSION TITLE
没有claudflarest.exe文件问题

### DISCUSSION THREAD

**Discussion by scout3096** [XIU2/CloudflareSpeedTest#528] [提问]
是我下载错误了吗
![屏幕截图 2024-07-21 010913](https://github.com/user-attachments/assets/d1ebf212-5327-4a87-95c4-7c1c53d74329)

**Comment by XIU2** [comment]
你自己都说出来了，这不是很明显么。。。

https://github.com/XIU2/CloudflareSpeedTest/releases/tag/v2.2.5

下载页不是标了 什么系统、什么架构、什么位数 分别要下载哪个。

**Reply by scout3096** [reply]
好的，感谢
